faker/providers/address/da_DK/__init__.py

```diff
@@ -6,41 +6,1719 @@ class Provider(AddressProvider):
     # Building numbers dont go higher than a 1000
     building_number_formats = ("%##", "%#", "%")
 
-    street_name_formats = ("{{street_prefix}} {{street_suffix}}",)
+    street_name_formats = ("{{street_prefix}}{{street_suffix}}",)
 
     street_address_formats = ("{{street_name}} {{building_number}}",)
 
     street_prefixes = (
-        "Gade",
-        "Allé",
-        "Parkere",
-        "Baggade",
-        "Gyde",
-        "Motorvej",
+        "Aberdeen",
```

Comment:
Those don't look like street prefixes. Maybe we should rename this property to `street_names`

Reply:
I took all street names from Copenhagen that end with the suffixes "gade" (street), "vej" (road) etc. and deleted the suffix part. So new street names can be generated from prefix and suffix.  Prefixes are usually the name of a plant, person, Nordic god, city etc.. If this is not in keeping with the terminology, I can change it?

Reply:
yes, let's change the name of the property.

Reply:
Done. But I had to change the method name to dk_street_name() as to not overwrite the street_name() method from the AddressProvider parent class.

Reply:
Why do you want to keep the logic from the parent method around? Can't you rename just the property, and the parent method would just work?